src/Provider/User/SymfonyUserProvider.php

```diff
@@ -10,21 +12,33 @@ final class SymfonyUserProvider implements UserProviderInterface
     /**
      * User for anonymous.
      */
-    const USER_ANONYMOUS = 'anonymous';
+    public const USER_ANONYMOUS = 'anonymous';
```

Comment:
private?

Reply:
Mark this const as private will introduce a BC break. But I think it's reasonable.